### DISCUSSION TITLE
cutomize Users

### DISCUSSION THREAD

**Discussion by barbaracrlp** [filamentphp/filament#12307] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hi I'm trying to make a panel for an existing DB , I already have a "usuarios" table where I want to get the users to access the panel.I'm encountering two problems: first I got different name for the columns like for example "mail" instead of "email, and second I got my own system of hashing the passwords, so I need Filament to get the password from my table and decrypted and compare with what the user is puting in the login form. Any Ideas? If someone could explain me please which steps should I do. I'm really new and I'm quite lost

**Comment by leandrocfe** [comment]
I believe it would be more efficient to modify your user table structure to match the default Laravel user structure...
